frontend/src/components/ICSUpload/ICSUpload.jsx

```diff
@@ -0,0 +1,52 @@
+import React, {useState} from 'react';
+
+const ICSUpload = ({onEventsImported}) => {
+    const [file, setFile] = useState(null);
+
+    const handleFileChange = (e) => {
+        setFile(e.target.files[0]);
```

Comment:
you dont need to create a new function for simple one-site call lines like this.